### ISSUE TITLE
One button to save all the current window's tabs in a new Collection

### ISSUE THREAD

**Issue by pacrob** [linkwarden/linkwarden#458]
**Is your feature request related to a problem? Please describe.**

I often have a number of tabs open that are related to whatever I'm project I'm currently working on. I'd like a quick way to save all those tabs so I can come back to them later.

**Describe the solution you'd like**

Click browser extension -> Click a "quick add all" (or something) button -> All tabs open in that window are saved to a new Collection titled with the date and time you clicked the button.

**Describe alternatives you've considered**

I can manually add each one, but it's a PITA. 

**Additional context**

I've used the OneTab extension for a while, it's good for temporarily reducing all your tabs, but not a good long-term solution.

**Comment by shufflerror** [comment]
Yeah would be great to have. maybe also the possibility to save all tabs to an existing collection

**Comment by daniel31x13** [comment]
Already implemented.

**Comment by mateitudor** [comment]
> Already implemented.

Could you provide a quick walkthrough? I cannot find this feature.

**Comment by Snuupy** [comment]
@mateitudor https://github.com/linkwarden/browser-extension/pull/113 looks like it's in context menu

The problem is that it doesn't specify a collection or a new collection to add, it just dumps all the tabs into "Uncategorized"
